src/waveform/renderers/allshader/waveformrendererhsv.cpp

```diff
@@ -69,10 +67,10 @@ void WaveformRendererHSV::paintGL() {
     const float breadth = static_cast<float>(m_waveformRenderer->getBreadth()) * devicePixelRatio;
     const float halfBreadth = breadth / 2.0f;
 
-    const float heightFactor = allGain * halfBreadth / 256.f;
+    const float heightFactor = allGain * halfBreadth / 255.f;
```

Comment:
I am in doubt this change is correct. At least 255 it is a "magic number" here. 
I think the code would be clearer if e change if we normalize maxAll to 1.0f 
What is the maxAll value maximum? 

The same applies to the other waveforms.

Reply:
Done